### PR TITLE
adding the packages section when reporting docker-image-scan results

### PR DESCRIPTION
**Pull request by itai1357**
adding the packages section when reporting docker-image-scan results

### CODE REVIEW

File: checkov/common/bridgecrew/vulnerability_scanning/integrations/docker_image_scanning.py [hunk 1]
```diff
@@ -48,6 +48,15 @@ def _create_report(
...
             for vul in twistcli_scan_result["results"][0].get("vulnerabilities") or []
         ]
+        packages = [
+            {
+                "type": package.get("type"),
+                "name": package.get("name"),
+                "version": package.get("version"),
+                "licenses": package.get("licenses") or [],
+            }
+            for package in twistcli_scan_result["results"][0].get("packages") or []
```
Comment (gruebel) [inline checkov/common/bridgecrew/vulnerability_scanning/integrations/docker_image_scanning.py:58]: can you create a new variable for `twistcli_scan_result["results"][0]` and use it here and above.

Comment (itai1357) [reply checkov/common/bridgecrew/vulnerability_scanning/integrations/docker_image_scanning.py:58]: sure!

**Review by nimrodkor — COMMENTED** [review]
@itai1357 Can you add some UTs for this?

**Review by nimrodkor — APPROVED** [review]
Nice!

**Pull request merged**